packages/admin/src/pages/super-admin-dashboard/Navigation.tsx

```diff
@@ -1,3 +1,5 @@
+'use client';
```

Comment:
Can likely fix this to render on the server if we replace the `userRouter` functionality - just didnt look much into this

Reply:
Resolved - cannot use a component that needs app router or pages router stuff in both app router & pages router, so opted for duplicating the component